js/src/modal/modal.spec.js

```diff
@@ -66,6 +260,7 @@ describe('Modal', () => {
       const originalPadding = '0px'
 
       document.body.style.paddingRight = originalPadding
+      document.body.appendChild(createTallContent())
```

Comment:
`should toggle a modal` test is created like that because most of the times a modal doesn't have a tall content.
What's the reason to add a tall content here?

Reply:
Within `should toggle a modal` it expects the original body padding to be stored in the data-padding-right attribute, which is only set when the body is overflowing. `should toggle a modal` can't both check the case that a modal doesn't have tall content, and also check that the original body padding is preserved.

Reply:
Ok make sense to me 👍